dev-docs/add-video-submodule.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: page_v2
+title: How to Add a Prebid.js Video submodule
+description: How to Add a Prebid.js Video submodule
+sidebarType: 1
+---
+
+# How to Add a Video Submodule
+{:.no_toc}
+
+Video submodules interact with the Video Module to integrate Prebid with Video Players, allowing Prebid to automatically:
+- render bids in the desired video player
+- mark used bids as won
+- trigger player and media events
+- populate the oRTB Video Impression and Content params in the bid request
+
+* TOC
+  {:toc }
+
+## Overview
+
+The Prebid Video Module simplifies the way Prebid integrates with video players by acting as a single point of contact for everything video.
+In order for the Video Module to connect to a video player, a submodule must be implemented. The submodule acts as a bridge between the Video Module and the video player.
+The Video Module will route commands and tasks to the appropriate submodule instance.
+A submodule is expected to work for a specific video player. i.e. the JW Player submodule is used to integrate Prebid with JW Player. The video.js submdule connects to video.js. 
+Publishers who use players from different vendors on the same page can use multiple video submodules.
+
+## Requirements
+
+The Video Module only supports integration with Video Players that meet the following requirements:
+- Must support parsing and reproduction of VAST ads
+  - Input can be an ad tag URL or the actual Vast XML.
```

Comment:
Extra space here can be removed. Would be good to be consistent and add periods to the end of the rest of these sentences.